gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -9383,13 +9474,43 @@ static PyObject* GemRB_SetPurchasedAmount(PyObject * /*self*/, PyObject* args)
 	Py_RETURN_NONE;
 }
 
+// a bunch of duplicated code moved from GemRB_ChangeStoreItem()
+static int SellBetweenStores(STOItem* si, int action, Store *store)
+{
+	CREItem ci = *reinterpret_cast<CREItem *>(si);
```

Comment:
We have a CopySTOItem method to do this — as far as it is possible, since not all the fields are shared.

Reply:
It's even integrated into a constructor!  I completely missed this.